docs/installation/auth/supervisor.md

```diff
@@ -44,6 +44,24 @@ Processes will be `STARTING`, `RUNNING`, or `ERROR`. If an error has occurred, c
  - celery workers: `log/worker.log`
  - celery beat: `log/beat.log`
  - authenticator: `log/authenticator.log`
+ 
+## Restarting Processes
+
+To restart the celery group:
+
+    sudo supervisorctl restart auth-celery:*
+
+To restart just celerybeat:
+
+    sudo supervisorctl restart auth-celerybeat
```

Comment:
Can you confirm this command actually works? In my experience when supervisor programs are grouped they have to be called in reference to the group, eg `auth-celery:auth-celerybeat` and not just `auth-celerybeat`

If that is the case, it would make more sense to call the group of processes `auth` and each process `celerybeat` or `celeryd`, that way the restart commands are `auth:celeryd` and `auth:celerybeat` or `auth:`